docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -40,6 +40,21 @@ The "type" field is the only mandatory field in a network object. For now there
 
 A command-line parameter named "default-network-template-path" of tenant controller will decide which default network will be created for a new space.
 
+The content of default network template file should reflect the Network object in json format, with ```{{.}}``` the replacement of tenant name, like
+```json
+{
+    "metadata": {
+        "name": "default",
+        "tenant": "{{.}}"
```

Comment:
is this a valid example?  I didn't see some required fields like "apiVersion" or "kind".  Also "tenant" and "finalizers" probably are not needed here.

Reply:
this is a valid example. the apiVersion & version will be the only option, which is arktos.futurewei.com and v1, respectively. In the future if multi versions are introduced, template content can have explicit GV to specify the desired one.

tenant template seems extraneous - it will always be filled with the right value by system. I'll make code change accordingly.
finalizers is put intentionally to remind that termination of network objects requires cleanup of other related resources.